src/tests/ftest/util/dmg_utils.py

```diff
@@ -28,19 +28,16 @@
 from pwd import getpwuid
 import re
 
-from command_utils_base import \
-    CommandFailure, FormattedParameter, CommandWithParameters, YamlParameters
-from command_utils import CommandWithSubCommand, YamlCommand
+from command_utils import \
```

Comment:
(pylint-no-name-in-module) No name 'FormattedParameter' in module 'command_utils'